PokeAlarm/Filters/EggFilter.py

```diff
@@ -46,6 +47,18 @@ def __init__(self, name, data):
             limit=BaseFilter.parse_as_set(
                 GymUtils.create_regex, 'gym_name_contains', data))
 
+        # Gym sponsor
+        self.gym_sponsor_index_contains = self.evaluate_attribute(
+            event_attribute='gym_sponsor', eval_func=GymUtils.match_regex_dict,
```

Comment:
`'gym_sponsor'` is parsed as an int - why are we comparing as a regex instead of a set of ints (like monster_id?)

Reply:
I do not know sorry. Code was taken verbatim from my fork where users have been using it for weeks, tidied up to appease Travis and submitted. Is it better to leave out the sponsor stuff and just submit the EX park stuff instead? As far as I know, nobody is using sponsor, but lots are using the park stuff to filter on EX raid eligibility

Reply:
The EX raid stuff is badly needed, the sponsor stuff less so.

Reply:
I believe Sponsor is valuable (at least in the US) as the majority of EX Raids are @ Sponsored Gyms in the US.   So I'd say both are equally important.  

I do think that if `sponsor` is just an id number, it would be better to compare this as int instead of as a regex.  I can help with converting this if you'd like